src/test/scala/faker/FakerSpec.scala

```diff
@@ -74,6 +74,12 @@ trait FakerSpec extends AnyFreeSpecLike with Checkers {
     testCanGen[address.StreetSuffix](locale, "address.StreetSuffix")(x =>
       Some(x.value)
     )
+    testCanGen[ancient.God](locale, "ancient.God")(x => Some(x.value))
```

Comment:
Looks good on these tests. We need another set of tests for the `Faker` class as well. See the Currency section here: https://github.com/etspaceman/scalacheck-faker/blob/master/src/test/scala/faker/FakerSpec.scala#L888